### PR TITLE
182 delete label

### PR DESCRIPTION
**Pull request by EkrlDev**

### CODE REVIEW

**Review by kod-man — CHANGES_REQUESTED** [review]

File: src/modals/AddItemModal.tsx [hunk 1]
```diff
@@ -56,13 +58,19 @@ const AddItemModal: FC<AddItemModalProps> = ({ tooltipLabel, onRefresh }) => {
...
+    tooltipLabel.slice(tooltipLabel.length, tooltipLabel.length);
 
   const submitHandler = () => {
     Axios.post(customAPI, { ...itemData, name: itemNameRef.current?.value })
       .then(() => {
         toast({
           ...defaultToastProps,
-          title: `${tooltipLabel} added successfully.`,
+          title:
+            tooltipLabel === "Project"
```
Comment (kod-man) [inline src/modals/AddItemModal.tsx:71]: If you use plural naming for project, dont need a check here

File: src/modals/AddItemModal.tsx [hunk 2]
```diff
@@ -76,11 +84,17 @@ const AddItemModal: FC<AddItemModalProps> = ({ tooltipLabel, onRefresh }) => {
       isFavorite: false
     });
     onClose();
-    onRefresh();
+    dispatch(refreshPage(tooltipLabel));
   };
   return (
     <>
-      <MyTooltip label={`Add ${tooltipLabel.toLowerCase()}`}>
+      <MyTooltip
```
Comment (kod-man) [inline src/modals/AddItemModal.tsx:91]: same as above

File: src/modals/AddItemModal.tsx [hunk 3]
```diff
@@ -76,11 +84,17 @@ const AddItemModal: FC<AddItemModalProps> = ({ tooltipLabel, onRefresh }) => {
...
+        label={
+          tooltipLabel === "Project"
```
Comment (kod-man) [inline src/modals/AddItemModal.tsx:93]: same as above

**Review by kod-man — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Components/DrawerComponent.tsx [hunk 4]
```diff
@@ -85,7 +79,7 @@ const DrawerComponent: FC<DrawerProps> = ({ isOpen, onClose }) => {
         >
           <Text color="gray">Projects</Text>
           <Flex>
-            <AddItemModal tooltipLabel="Project" onRefresh={refreshPage} />
+            <AddItemModal tooltipLabel="Project" />
```
[the remark this answers is not in the document]
Comment (EkrlDev) [reply src/components/Navbar/Components/DrawerComponent.tsx:82]: label={
          tooltipLabel === "Project"
            ? `Add ${tooltipLabel.toLowerCase()}` ===> last letter is not deleted
            : `Add new ${title.toLowerCase()}` ==> last letter deleted
        }
last chars are not deleted if its a "Project" label

[the remark this answers is not in the document]
Comment (kod-man) [reply src/components/Navbar/Components/DrawerComponent.tsx:82]: Why do we need to check project?

**Review by kod-man — APPROVED** [review]

**Pull request merged**